nodebuilder/node/admin.go

```diff
@@ -19,26 +18,17 @@ func newAdmin(tp Type) Module {
 	}
 }
 
-func (a *admin) Type(context.Context) Type {
-	return a.tp
+// Info contains information related to the administrative
+// node.
+type Info struct {
+	Type       Type   `json:"type"`
+	APIVersion string `json:"api_version"`
 }
 
-// Version represents all binary build information.
-type Version struct {
-	SemanticVersion string `json:"semantic_version"`
-	LastCommit      string `json:"last_commit"`
-	BuildTime       string `json:"build_time"`
-	SystemVersion   string `json:"system_version"`
-	GoVersion       string `json:"go_version"`
-}
-
-func (a *admin) Version(context.Context) Version {
-	return Version{
-		SemanticVersion: semanticVersion,
-		LastCommit:      lastCommit,
-		BuildTime:       buildTime,
-		SystemVersion:   fmt.Sprintf("%s/%s", runtime.GOARCH, runtime.GOOS),
-		GoVersion:       runtime.Version(),
+func (a *admin) AdminInfo(context.Context) Info {
```

Comment:
Just Info, like the struct? Admin prefix feels like there is another less sensitive Info. Also, all methods on the Node struct are sort of an admin. Wait, actually, Info can be read or public. There is nothing wrong with API users knowing what API version they depend on and what node type they are connected to.

Reply:
Can't do `Info`. I wish we could, but prevented by `client.API`.